pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers=[
 ]
 dependencies = [
     "torch",
-    "qadence[braket, pulser, horqrux]",
+    "qadence[pulser, horqrux]==1.7.8",
```

Comment:
Still unsure about pinning the versions.

Reply:
I'll fix it in another MR as we have changed to using Noise handler instead of Noise in qadence